napari/layers/utils/_tests/test_layer_utils.py

```diff
@@ -77,25 +77,21 @@ def test_calc_data_fast_uint8():
     assert calc_data_range(data) == [0, 255]
 
 
-@pytest.mark.timeout(2)
```

Comment:
Do you want to add back here too? `calc_data_range` was the function giving us most of the "slow" behaviour before. probably worth having at this level

Reply:
added, & parameterized